packages/backend/src/api/controllers/tvl/DetailedTvlController.ts

```diff
@@ -25,6 +25,30 @@ import {
 } from './detailedTvl'
 import { generateDetailedTvlApiResponse } from './generateDetailedTvlApiResponse'
 
+interface DetailedControllerBehaviorFlags {
+  skipUnsyncedDetailedTvl: boolean
```

Comment:
Propagate this new, better name through here

Reply:
https://github.com/l2beat/l2beat/pull/1776/commits/c2e508e7976519fcbc105e390e3688848bf87989